### ISSUE TITLE
[Bug] Chinese input method is not supported well in Windows

### ISSUE THREAD

**Issue by chenyukang** [AppFlowy-IO/AppFlowy#1756]
### Bug Description

I can not input Chinese words correctly in Microsoft Pinyin (verified on full-pinyin and Double pinyin).

![demo](https://user-images.githubusercontent.com/230646/215334611-11b76dcd-fc9b-411f-9d92-aec865a207aa.gif)

I remember Obsidian has similiar issue, which caused by CodeMirror, update to CodeMirror 6.0 resolved it.
https://forum.obsidian.md/t/chinese-ime-issues-in-windows-1909/30018/9

Not sure whether AppFlowy uses CodeMirror, this issue seems more serious, since we can not input anything in Chinese.

### How to Reproduce

Open any document and begin to type in Pinyin.

### Expected Behavior

Input correctly.

### Operating System

Windows 11 Enterprise

### AppFlowy Version(s)

v0.0.9.1

**Comment by annieappflowy** [comment]
Thanks for reporting.
This is a duplicate of https://github.com/AppFlowy-IO/AppFlowy/issues/1517

**Comment by 6ag** [comment]
@chenyukang 而且输入法候选框不能正确的跟随光标位置，这个好像是Flutter一直存在的硬伤

**Comment by jwcen** [comment]
@annieappflowy Has It been fixed? This bug is so annoying that I can't work

**Comment by annieappflowy** [comment]
Hi @jwcen , this is a known issue, but we have no control over it. We've reported it to the Google Flutter team: https://github.com/flutter/flutter/issues/113909#issuecomment-1311478223, which has not been fixed. Waiting on the fix, we are working on the web app using a different framework, which won't have any issues typing in simplified or traditional Chinese characters.

**Comment by leenzhu** [comment]
Just happend to find this issue https://github.com/flutter/flutter/issues/113909#issuecomment-1311478223 has been closed this as [completed](https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aclosed+archived%3Afalse+reason%3Acompleted).

FYI. :-)

**Comment by annieappflowy** [comment]
Thanks @leenzhu for the heads-up. Checking

**Comment by Renzo-Olivares** [comment]
Hello the issue closed in https://github.com/flutter/flutter/issues/113909#issuecomment-1311478223 was related to a similar issue on Linux. I don't think this will solve this issue on Windows. I would be interested in knowing more about this issue so I can help solve it if possible. I am unable to reproduce this issue on the flutter/samples demo of TextEditingDeltas. https://github.com/flutter/samples/tree/main/simplistic_editor using `Chinese (Simplified, China) Microsoft Pinyin`.

**Comment by annieappflowy** [comment]
Thanks @Renzo-Olivares for following up with us. We're still checking. Will update on this thread

**Comment by annieappflowy** [comment]
Fixed. Please give it a try with the latest [release](https://github.com/AppFlowy-IO/AppFlowy/releases) and let us know if you run into any issues.
<img width="322" alt="image" src="https://github.com/AppFlowy-IO/AppFlowy/assets/12026239/25c85611-db95-4bcf-b616-feceae565fcf">

**Comment by jinshaopu** [comment]
0.3.7   on ubuntu 23.04 with fctix5 .the same issues

**Comment by toplinuxsir** [comment]
0.4.0 on ubuntu 23.04 with fcitx5 . the same issues!

**Comment by lynnww** [comment]
0.4.3 repeat the bug again.

![image](https://github.com/AppFlowy-IO/AppFlowy/assets/7655742/978c433c-6c01-4334-8388-a2d4a56707b0)

**Comment by MyEclipseyang** [comment]
> 0.4.3 repeat the bug again.
> 
> ![image](https://private-user-images.githubusercontent.com/7655742/298447030-978c433c-6c01-4334-8388-a2d4a56707b0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NTY2NTEsIm5iZiI6MTcwNjg1NjM1MSwicGF0aCI6Ii83NjU1NzQyLzI5ODQ0NzAzMC05NzhjNDMzYy02YzAxLTQzMzQtODM4OC1hMmQ0YTU2NzA3YjAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMDJUMDY0NTUxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWU3MGQzYjA2OWE0ZTk3N2ExYjdlY2JlMmExMWFmYWEwZmY1ODY4ODM4ZTcyYzVkMGRhMDQ0MGFjOGNiODk5NCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.jVin4-b_xcd-sIAEvXRwyPLda6uCxDGHPihFkLh_82s)

So did I

**Comment by MyEclipseyang** [comment]
> 0.4.3 repeat the bug again.
> 
> ![image](https://private-user-images.githubusercontent.com/7655742/298447030-978c433c-6c01-4334-8388-a2d4a56707b0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NTY2NTEsIm5iZiI6MTcwNjg1NjM1MSwicGF0aCI6Ii83NjU1NzQyLzI5ODQ0NzAzMC05NzhjNDMzYy02YzAxLTQzMzQtODM4OC1hMmQ0YTU2NzA3YjAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMDJUMDY0NTUxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWU3MGQzYjA2OWE0ZTk3N2ExYjdlY2JlMmExMWFmYWEwZmY1ODY4ODM4ZTcyYzVkMGRhMDQ0MGFjOGNiODk5NCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.jVin4-b_xcd-sIAEvXRwyPLda6uCxDGHPihFkLh_82s)

I update to 0.4.5, and find AppFlowy can enter Chinese correctly

**Comment by xiwwqk** [comment]
0.53再次出现错误，使用微软输入法正常输入，使用其他输入法会出现错误，但在一行开头加入空格之后，就会正常输入。
